src/Microsoft.Health.Dicom.Core.UnitTests/Features/Validation/DicomDateValidationTests.cs

```diff
@@ -0,0 +1,34 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using Dicom;
+using Microsoft.Health.Dicom.Core.Exceptions;
+using Microsoft.Health.Dicom.Core.Features.Validation;
+using Xunit;
+
+namespace Microsoft.Health.Dicom.Core.UnitTests.Features.Validation
+{
+    public class DicomDateValidationTests
```

Comment:
Brandon removed Dicom from all class name prefix, because as a team we thought it was reduandent to use Dicom in Dicom project.

Please remove Dicom prefix in all classed